eng/build.ps1

```diff
@@ -34,6 +34,15 @@ Param(
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 
+# Workaround for DOTNET_HOST_PATH not being set by older MSBuild
+# Removal is tracked by https://github.com/microsoft/testfx/issues/6767
+if (-not $env:DOTNET_HOST_PATH) {
```

Comment:
Note that our CI sometimes uses globally installed SDK so we needed to tweak the logic a bit: https://github.com/dotnet/roslyn/pull/80842, I'm not sure if that's something you need to do as well